src/Microsoft.AspNet.Razor/Parser/SyntaxTree/Span.cs

```diff
@@ -126,29 +126,32 @@ public override bool EquivalentTo(SyntaxTreeNode node)
         {
             var other = node as Span;
             return other != null &&
-                   Kind.Equals(other.Kind) &&
-                   Start.Equals(other.Start) &&
-                   EditHandler.Equals(other.EditHandler) &&
-                   string.Equals(other.Content, Content, StringComparison.Ordinal);
+                Kind.Equals(other.Kind) &&
+                Start.Equals(other.Start) &&
+                EditHandler.Equals(other.EditHandler) &&
+                string.Equals(other.Content, Content, StringComparison.Ordinal);
```

Comment:
Hmm curious as to why we never added the `CodeGenerator` comparison here. /cc @anurse